Sources/SwiftKueryPostgreSQL/PostgreSQLParameterSet.swift

```diff
@@ -23,7 +23,7 @@ internal struct PostgreSQLParameterSet {
     /// - Throws: QueryError.unsupported if parameter cannot be converted to data
     internal func withUnsafeBufferPointers(_ body: @escaping (UnsafePointers) -> Void) throws {
         let (values, lengths, formats) = try parameterData()
-        defer { values.forEach({ free($0) }) }
+        defer { values.compactMap({ $0 }).forEach({ free($0) }) }
```

Comment:
Was there a problem that the compactMap is solving?

Reply:
Yes. `values` is an array of optionals, and the `free` function expects a non-optional (at least in the latest compiler/library versions). This code would not compile with my Swift 5.5 toolchain without replacing forEach with compactMap, to remove the optionality.